llvm/include/llvm/Analysis/IR2Vec.h

```diff
@@ -53,7 +56,12 @@ class raw_ostream;
 enum class IR2VecKind { Symbolic };
 
 namespace ir2vec {
-/// Embedding is a datavtype that wraps std::vector<double>. It provides
+
+LLVM_ABI extern cl::opt<float> OpcWeight;
```

Comment:
Why do these need an `LLVM_ABI` tag?

If this is actually needed, this seems like something that should be split out of this patch.